src/shim.js

```diff
@@ -176,20 +176,14 @@ function syncOutObjects(objects) {
 }
 
 /**
- * Executes a command by first checking if it is a custom method and then calling the method on the target.
+ * Executes a command.
  * @param command the command to execute
  */
 function executeCommand(command) {
     if (commands[command.name]) {
         return commands[command.name](command);
```

Comment:
Trying to figure out what happened here? How is the `objectSpace` being looked up now?